inspect/inspect_test.go

```diff
@@ -0,0 +1,163 @@
+package inspect_test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/fortytw2/leaktest"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	abci_types "github.com/tendermint/tendermint/abci/types"
```

Comment:
`abcitypes` and so on below (cf. https://golang.org/doc/effective_go#package-names re: underscores in package names/aliases)